src/Entity/Donation.php

```diff
@@ -22,6 +25,13 @@ class Donation implements GeoPointInterface
     use EntityPostAddressTrait;
     use EntityPersonNameTrait;
 
+    public const STATUS_WAITING_CONFIRMATION = 'waiting_confirmation';
```

Comment:
Just my thought, no need to change but it was ok with the enum class.

Reply:
Enum class was overkill, no methods in enum was used.